src/app/classes/external-references.ts

```diff
@@ -205,41 +213,58 @@ export class ExternalReferences extends Serializable {
    */
   public parseCitations(
     value: string,
-    restApiConnector: RestApiConnectorService
+    restApiConnector: RestApiConnectorService,
+    field?: string
```

Comment:
It looks like the new `field` parameter is never used in this function, it should be removed if it's not necessary